app/controllers/api/v0/webhooks_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Api
+  module V0
+    class WebhooksController < ApiController
+      respond_to :json
+      before_action :authenticate!
+
+      def create
+        @webhook = current_user.webhook_endpoints.create!(webhook_params)
+        render "show", status: :created
+      end
+
+      def show
+        @webhook = Webhook::Endpoint.includes(:user).find(params[:id])
+      end
+
+      def destroy
+        webhook = current_user.webhook_endpoints.find(params[:id])
+        webhook.destroy
+        render json: { success: webhook.destroyed? }
```

Comment:
According to HTTP specs if a DELETE is not successful a non successful status code should be return, so this is potentially misleading to a client that sees a HTTP 200 OK but gets a false in the body because the object destruction has failed. I'll submit a PR for this